test/splitter.js

```diff
@@ -21,11 +21,11 @@ contract('Splitter', async accounts => {
         );
     }
 
-    const [ contractOwner, senderAddress, recipientOneAddress, recipientTwoAddress] = accounts;
+    const [ contractOwnerAddress, senderAddress, recipientOneAddress, recipientTwoAddress] = accounts;
```

Comment:
Meh, you could save some screen space by removing all these `Address` suffixes.

Reply:
I can sometimes take Uncle Bob Martin's descriptive variable names too far!